eth/downloader/downloader.go

```diff
@@ -99,6 +94,26 @@ var (
 	errTooOld                  = errors.New("peer doesn't speak recent enough protocol version (need version >= 62)")
 )
 
+func ErrWasRequested(e error) bool {
```

Comment:
This is mostly a convenience function for being able to handle logging verbosities and values with a little more descriptiveness and precision.